Assets/Scripts/View/ToolBar/Timer.cs

```diff
@@ -22,42 +25,49 @@ public IEnumerator StartTimer(int time)
         {
             CurrentTime = time;
             GameState state = controller.game.State.CurrentState;
+            var last_tick = PhotonNetwork.Time;
```

Comment:
Ooo!  🌟 

I was curious how will you handle time because it gets  tricky over network. This is a nice implementation!